lib/open_food_network/option_value_namer.rb

```diff
@@ -72,5 +71,39 @@ def scale_for_unit_value
 
       unit
     end
+
+    def pluralize(unit_name, count)
+      I18nUnitNames.instance.pluralize(unit_name, count)
+    end
+
+    # Provides efficient access to unit name inflections.
+    # The singleton property ensures that the init code is run once only.
+    # The OptionValueNamer is instantiated in loops.
+    class I18nUnitNames
```

Comment:
I think this break the single responsibility rule. We should have this in a separate class.
UnitNamePlural or Pluralizer or I18nUnitNames.
This (including option_value_names) should all go to something like app/services/naming

Reply:
I agree, I will move it.